Tools/General tool list.md

```diff
@@ -89,17 +67,7 @@ An extension for easily creating tab screencasts in Google Chrome.
 Used for transferring files between an Android phone and a macOS system. Get it [here](https://www.android.com/filetransfer/).
 
 ### Android Debug Bridge (adb)
-
 Used for communicating with your Android device via the terminal.
-### Chuck and Loggie
```

Comment:
Not sure if we need to remove these. It is created by Inifnum devs and I think they might still be used.

Reply:
Thanks for clarification. If deprecated, then we should remove them